caom2-tap/src/main/java/ca/nrc/cadc/caom2ops/CadcMastCutoutGenerator.java

```diff
@@ -81,44 +84,32 @@
  */
 public class CadcMastCutoutGenerator extends CadcMastResolver implements CutoutGenerator {
     private static final Logger log = Logger.getLogger(CadcMastCutoutGenerator.class);
+    private final CutoutGenerator cadcCutoutGenerator = new CadcCutoutGenerator();
```

Comment:
This needs to be a CadcCutoutGenerator("mast") ??

Reply:
Fixed.